platforms/r3-corda-ent/charts/node/templates/deployment.yaml

```diff
@@ -388,6 +393,7 @@ spec:
               echo
               sleep ${HOW_LONG}
           fi
+          sleep 1200
```

Comment:
This can be removed

Reply:
* Sleep deleted.
* For the testing, a corda-ent network was deployed (cenm v1.5) with 3 node orgs ( no firewall and no cordapp). Validated using the endpoint /network-map-user/node-infos that the 2 notaries and 3 nodes were registered.